open_bus_stride_api/routers/siri_vehicle_locations.py

```diff
@@ -67,6 +67,7 @@ def list_(limit: int = None, offset: int = None,
           order_by: str = None, siri_routes__line_ref: str = None, siri_routes__operator_ref: str = None,
           siri_rides__schedualed_start_time_from: datetime.datetime = None,
           siri_rides__schedualed_start_time_to: datetime.datetime = None,
+          siri_rides__id: int = None, siri_routes__id: int = None,
```

Comment:
should be `siri_rides__ids` / `siri_routes__ids` - to be similar to other id fields and allow to request multiple ids at once